dexmaker/src/main/java/external/com/android/dx/DexMaker.java

```diff
@@ -702,6 +710,10 @@ boolean isDirect() {
         }
 
         EncodedMethod toEncodedMethod(DexOptions dexOptions) {
+            if ((flags & ABSTRACT) != 0 || (flags & NATIVE) != 0) {
```

Comment:
I think dexmaker is just for hooker generation, what's the point of checking native flags here?

Reply:
I believe that native methods don’t have bodies so they should be handled like abstract methods. Anyways, I imported latest changes to be more in line with original dexmaker, so I don't believe this change would do any harm.

Reply:
I believe that native methods don’t have bodies so they should be handled like abstract methods. Anyways, I imported missing changes to be more in line with original dexmaker, so I don't believe this change would do any harm.